tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -83,6 +86,8 @@ spec:
       # TODO: Currently pathing the prometheus manifests for remote write. Move this to upsteam going forward.
       echo "volumeBindingMode: WaitForFirstConsumer" >> $(workspaces.source.path)/perf-tests/clusterloader2/pkg/prometheus/manifests/0ssd-storage-class.yaml
       cat $(workspaces.source.path)/perf-tests/clusterloader2/pkg/prometheus/manifests/0ssd-storage-class.yaml
+      sed -i '/nodeSelector/d' $(workspaces.source.path)/perf-tests/clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml
```

Comment:
Please add comment explaining what you are trying to do here.

Reply:
Please add these comment in the file for posterity.
We want other people not just 3 of us to understand why we are doing it when they look at this file.
We should be able to understand it after we come back months later.

Reply:
yeah added.